components/src/StoriesForTests/__snapshots__/Select.story.storyshot

```diff
@@ -5259,7 +5259,7 @@ exports[`Storyshots StoriesForTests/Select Controlled 1`] = `
                                                                 autoCorrect="off"
                                                                 className="SelectTest__input"
                                                                 disabled={null}
-                                                                id="react-select-26-input"
+                                                                id="react-select-27-input"
```

Comment:
I don't get why this id is changing ??? 😕

Reply:
me neither 😞